docs/05-ingest.md

```diff
@@ -155,8 +159,64 @@ There are several stacks that allow data to be imported using the bulk import pr
   as Fargate tasks. Future work will allow them to run on EC2 instances. This stack is experimental.
 
 These can all be deployed independently of each other. Each stack has its own queue from which it pulls jobs. The
-`sleeper.optional.stacks` instance property needs to include `EmrBulkImportStack`, `PersistentEmrBulkImportStack` or
-`EksBulkImportStack` respectively.
+`sleeper.optional.stacks` instance property needs to include `EmrServerlessBulkImportStack`, `EmrBulkImportStack`, `PersistentEmrBulkImportStack` or `EksBulkImportStack` respectively.
+
+#### Bulk import on EMR Serverless
+
+The EMR Serverless stack creates an EMR Serverless application that only runs when there are jobs to process. 
+When you want to run a job the application is started by EMR Serverless. After 15 minutes of inactivity the application 
+is shutdown ready to be started when needed. 
+
+A simple example of the message to send is:
+
+```JSON
+{
+  "tableName": "my-table",
+  "files": [
+    "my-bucket/my-files/"
+  ]
+}
+```
+
+This message needs to be sent to the queue with URL given by the value of the
+property `sleeper.bulk.import.emr.serverless.job.queue.url`.
+
+When you submit your JSON job via the SQS Queue, an EMR Serverless job should appear in the application found in 
+the EMR Studio part of the AWS console with your desired 
+configuration. Once the job starts (around 2 minutes), you will be able to follow the links in EMR Studio to access your Spark UI. 
+This will allow you to monitor your job and view logs from the Spark executors and driver. You can also access previous job Spark UI's from EMR Studio. 
+After your job finishes the application with auto shutdown after 15 minutes. When in the stopped state it takes seconds for the application to start when a new job is received.
+
+The following can be edited in the Sleeper Admin console. With the
+current stack implementation it is not possible to configure these on a job basis. 
```

Comment:
Just linking this to the issue to change that: https://github.com/gchq/sleeper/issues/1138